### PR TITLE
move upgrading db modules from cluster to db page

### PR DESCRIPTION
**Pull request by AlonMagrafta**

### CODE REVIEW

**Comment by bmansheim** [comment]
@AlonMagrafta Good thinking. Maybe we should have a separate upgrading modules page that includes upgrading on the cluster and the db. What do you say?

**Comment by AlonMagrafta** [comment]
Sounds good too!

File: content/modules/add-module-to-database.md [hunk 1]
```diff
@@ -43,3 +43,84 @@ To add modules to a database:
...
+    ![module_info-1](/images/rs/module_info-1.png?width=1000&height=382)
+
+1. To see the versions of the modules on the cluster, run either:
+
+    - `rladmin status modules` - Shows the latest modules available on the cluster
+        and the modules used by databases.
+    - `rladmin status modules all` - Shows all of the modules available on the cluster
+        and the modules used by databases.
+
+1. To upgrade a database to the latest version of Redis
```
Comment (kaitlynmichael) [inline content/modules/add-module-to-database.md:80]: If they can upgrade the database and its modules all at once, why would they need the previous few steps to find the module name and version?

File: content/modules/add-module-to-database.md [hunk 2]
```diff
@@ -43,3 +43,84 @@ To add modules to a database:
 
 1. Click **Show advanced options** and enter a port number for the database, such as: **12543**.
 1. Click **Activate**.
+
+
+## Upgrading the module for the database
+
+{{< note >}}
+After you upgrade the module for a database, the database shards are restarted.
+This causes a short interruption in the availability of this database across the cluster.
```
Comment (kaitlynmichael) [inline content/modules/add-module-to-database.md:52]:
```suggestion
```

File: content/modules/add-module-to-database.md [hunk 3]
```diff
@@ -43,3 +43,84 @@ To add modules to a database:
...
+{{< /note >}}
+
+To upgrade a module that is installed on a cluster:
+
+1. Connect to the terminal of a node in the cluster
```
Comment (kaitlynmichael) [inline content/modules/add-module-to-database.md:57]:
```suggestion
```

File: content/modules/add-module-to-database.md [hunk 4]
```diff
@@ -43,3 +43,84 @@ To add modules to a database:
...
+1. Run `rladmin status` to list the databases on the node.
```
Comment (kaitlynmichael) [inline content/modules/add-module-to-database.md:58]:
```suggestion
1. From the terminal, run `rladmin status` to list the databases on the node.
```

File: content/modules/add-module-to-database.md [hunk 5]
```diff
@@ -43,3 +43,84 @@ To add modules to a database:
...
+
+    1. Extract the module archive (zip) file.
+    1. Open the JSON file.
+    1. Find the module name and version number in the file.
+
+    An example of the JSON file for the RediSearch module is:
+
+    ![module_info-1](/images/rs/module_info-1.png?width=1000&height=382)
+
+1. To see the versions of the modules on the cluster, run either:
```
Comment (kaitlynmichael) [inline content/modules/add-module-to-database.md:73]: Is this an alternative to the previous step, or are both necessary?

File: content/modules/add-module-to-database.md [hunk 6]
```diff
@@ -43,3 +43,84 @@ To add modules to a database:
...
+
+1. Connect to the terminal of a node in the cluster
+1. Run `rladmin status` to list the databases on the node.
+1. Copy the name of the database that uses the module that you want to upgrade.
+
+    ![rladmin_status-1](/images/rs/rladmin_status-1.png?width=1000&height=214)
+
+1. Find the exact module name and version:
+
+    1. Extract the module archive (zip) file.
```
Comment (kaitlynmichael) [inline content/modules/add-module-to-database.md:65]: Where is this file located?

File: content/modules/add-module-to-database.md [hunk 7]
```diff
@@ -43,3 +43,84 @@ To add modules to a database:
 
 1. Click **Show advanced options** and enter a port number for the database, such as: **12543**.
 1. Click **Activate**.
+
+
+## Upgrading the module for the database
```
Comment (kaitlynmichael) [inline content/modules/add-module-to-database.md:48]: Can this only be done from the terminal with `rladmin` or is there an option to do this from the API and/or admin console?

**Review by kaitlynmichael — CHANGES_REQUESTED** [review]

**Comment by kaitlynmichael** [comment]
this work is being tracked by https://redislabs.atlassian.net/browse/DOC-686

**Comment by kaitlynmichael** [comment]
I'm going to push this through just so the page gets moved and open a new PR for my rewrites (DOC-686)

File: content/rs/release-notes/rs-6-0-8-september-2020.md [hunk 8]
```diff
@@ -46,7 +46,7 @@ The following GA releases of Redis Modules are bundled in RS 6.0:
 - [RedisTimeSeries](https://redislabs.com/redis-enterprise/redis-time-series/), version [1.2.7]({{< relref "/modules/redistimeseries/release-notes/redistimeseries-1.2-release-notes.md" >}}) (updated)
 - [RedisBloom](https://redislabs.com/redis-enterprise/redis-bloom/), version [2.2.4]({{< relref "/modules/redisbloom/release-notes/redisbloom-2.2-release-notes.md" >}}) (updated)
 
-To use the updated modules with a database, you must [upgrade the module on the database]({{< relref "/modules/add-module-to-cluster#upgrading-the-module-for-the-database" >}}).
+To use the updated modules with a database, you must [upgrade the module on the database]({{< relref "/modules/add-module-to-database/#upgrading-the-module-for-the-database" >}}).
```
Comment (kaitlynmichael) [inline content/rs/release-notes/rs-6-0-8-september-2020.md:49]:
```suggestion
To use the updated modules with a database, you must [upgrade the module on the database]({{< relref "/modules/add-module-to-database.md" >}}).
```

File: content/rs/release-notes/rs-6-0-8-september-2020.md [hunk 9]
```diff
@@ -34,7 +34,7 @@ For more information about Redis 6.0.5, check out the [release notes](https://ra
 
 The [rladmin CLI](https://docs.redislabs.com/latest/rs/references/rladmin/) introduces several updates to the commands for upgrading modules.
 It is now easier to upgrade your modules to the latest module version.
-Find out more [here](https://docs.redislabs.com/latest/modules/add-module-to-cluster/#upgrading-the-module-for-the-database).
+Find out more [here](https://docs.redislabs.com/latest/modules/add-module-to-database/#upgrading-the-module-for-the-database).
```
Comment (kaitlynmichael) [inline content/rs/release-notes/rs-6-0-8-september-2020.md:37]:
```suggestion
Find out more [here]({{< relref "/modules/add-module-to-database.md" >}}).
```

File: content/modules/add-module-to-cluster.md [hunk 10]
```diff
@@ -45,83 +45,6 @@ To deploy an upgraded package:
...
-    rladmin upgrade module db_name MyDB module_name ReJSON version 10004 module_args keep_args and module db_name MyDB module_name bf version 20201 module_args ""
-    ```
-
-- To upgrade the database to use the latest version of Redis and use the latest version of the modules that are used by the database:
-
-    ```sh
-    rladmin upgrade db shopping_cart latest_with_modules
-    ```
+[Adding modules to a database]({{< relref "/modules/add-module-to-database.md">}}).
+[Upgrading a database's module]({{< relref "/modules/add-module-to-database/#upgrading-the-module-for-the-database">}}).
```
Comment (kaitlynmichael) [inline content/modules/add-module-to-cluster.md:50]:
```suggestion
[Upgrading a database's module]({{< relref "/modules/add-module-to-database.md">}}).
```

File: content/modules/add-module-to-database.md [hunk 11]
```diff
@@ -43,3 +43,84 @@ To add modules to a database:
...
+
+{{< note >}}
+After you upgrade the module for a database, the database shards are restarted.
+This causes a short interruption in the availability of this database across the cluster.
+{{< /note >}}
+
+To upgrade a module that is installed on a cluster:
```
[the remark this answers is not in the document]
Comment (AlonMagrafta) [reply content/modules/add-module-to-database.md:54]: @kaitlynmichael think about the modules that are loaded to the cluster as the repository for modules you can create a DB with, on this cluster

File: content/modules/add-module-to-database.md (hunk 6 above)
Comment (AlonMagrafta) [reply content/modules/add-module-to-database.md:63]: You are downloading it from our download center.

File: content/modules/add-module-to-database.md (hunk 5 above)
Comment (AlonMagrafta) [reply content/modules/add-module-to-database.md:71]: that's different.
one is to see what module version is the file, the other is to get the list of modules and their versions that are loaded to the cluster

File: content/modules/add-module-to-database.md (hunk 1 above)
Comment (AlonMagrafta) [reply content/modules/add-module-to-database.md:78]: to make sure you upgrade to the module version you wanted to

**Pull request closed without merging**